src/Filters/Has.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace LaravelJsonApi\Eloquent\Filters;
+
+use LaravelJsonApi\Eloquent\Contracts\Filter;
+use LaravelJsonApi\Eloquent\Schema;
+
+class Has implements Filter
+{
+    use Concerns\DeserializesValue;
+    use Concerns\IsSingular;
+
+    /**
+     * @var string
+     */
+    private string $relationName;
```

Comment:
Rename to `$fieldName`, to make it clearer we are referring to the JSON:API field name.